pkgs/build-support/composer/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchComposer }:
+
+{ name ? "${args.pname}-${args.version}"
+, src ? null
+, srcs ? null
+, composerSha256 ? "unset"
+, composerPatches ? []
+, ...
+} @ args:
+
+let
+  composerDeps = fetchComposer {
+    inherit name src srcs;
```

Comment:
I do not like inheriting just a subset of args. Just today I had issues with `buildGoModule` not responding to `postPatch` for the `deps` derivation.

Reply:
I've added a `composerAttrs` argument that can be used to pass arbitrary attributes to the `fetchComposer` derivation.